MinPlatformPkg/PlatformInit/PlatformInitSmm/PlatformInitSmm.c

```diff
@@ -125,39 +147,8 @@ PlatformInitSmmEntryPoint (
   IN EFI_SYSTEM_TABLE *SystemTable
   )
 {
-  EFI_STATUS Status;
-  VOID       *SmmEndOfDxeRegistration;
-  VOID       *SmmReadyToLockRegistration;
-  VOID       *SmmReadyToBootRegistration;
-  VOID       *SmmExitBootServicesRegistration;
-
-  Status = gSmst->SmmRegisterProtocolNotify (
-                    &gEfiSmmEndOfDxeProtocolGuid,
-                    SmmEndOfDxeEventNotify,
-                    &SmmEndOfDxeRegistration
-                    );
-  ASSERT_EFI_ERROR (Status);
-  
-  Status = gSmst->SmmRegisterProtocolNotify (
-                    &gEfiSmmReadyToLockProtocolGuid,
-                    SmmReadyToLockEventNotify,
-                    &SmmReadyToLockRegistration
-                    );
-  ASSERT_EFI_ERROR (Status);
-  
-  Status = gSmst->SmmRegisterProtocolNotify (
-                    &gEdkiiSmmReadyToBootProtocolGuid,
-                    SmmReadyToBootEventNotify,
-                    &SmmReadyToBootRegistration
-                    );
-  ASSERT_EFI_ERROR (Status);
-  
-  Status = gSmst->SmmRegisterProtocolNotify (
-                    &gEdkiiSmmExitBootServicesProtocolGuid,
-                    SmmExitBootServicesEventNotify,
-                    &SmmExitBootServicesRegistration
-                    );
-  ASSERT_EFI_ERROR (Status);
+  PlatformInitMmEntryPoint ();
 
+  // Always succeeds
   return EFI_SUCCESS;
-}
```

Comment:
nit: new line at the end of a file is missing.

Reply:
I looked at the file and it does have a new line at the end of the file.  I think this is just a weird github PR GUI thing where it shows that it previously didn't have a new line at the end of file (hence why it is only in the deleted line).